tests/end2end/make-test.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+# Make a test skeleton.
+
+TEST_NAME=${1}
+
+if [[ -z ${TEST_NAME} ]]; then
+    echo "usage: make-test.sh 'test-name'"
+    exit 1
+fi
+
+echo "Making skeleton for a test named ${TEST_NAME}"
+mkdir ${TEST_NAME}
+pushd ${TEST_NAME} > /dev/null
+
+cat <<EOF > test.sh
+#!/usr/bin/env bash
+# ${TEST_NAME} test: TODO Add description of test.
+
+source ../utils.sh
+
+set -e
+
+test_init
+EOF
+
+cat <<EOF > clean.sh
+#!/usr/bin/env bash
+rm -rf .git > /dev/null 2>&1
+rm -rf .gitattributes > /dev/null 2>&1
+rm *.pt > /dev/null 2>&1
+EOF
```

Comment:
If clean.sh is the same for all tests, should we just put it in tests/end2end and have it called one level up?

Reply:
`clean.sh` can be different for different test (i.e. in smuge is also deletes some json files) so currently we need to have per-test scripts. We could make the runner check for the existence of a test-specific but the clean commands are so small it doesn't seem like a big deal right now.